### PR TITLE
Add Header with Buttons and SVG

### LINKED ISSUE #4: As a user, I want to see a Header where I interact with the app
- [ ] Switch Button Card
- [ ] Switch Button List

### PR DESCRIPTION
**Pull request by franzosterkamp**
Bildschirmfoto 2019-12-18 um 17 06 46

closes #4

### CODE REVIEW

File: client/.storybook/addons.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import "@storybook/addon-actions/register";
-import "@storybook/addon-links/register";
-import "@storybook/addon-knobs";
-import "@storybook/addon-viewport";
+import '@storybook/addon-actions/register';
+import '@storybook/addon-links/register';
+import '@storybook/addon-knobs/register';
+import '@storybook/addon-viewport/register';
```
Comment (aruppert) [inline client/.storybook/addons.js]: that's the prettifying issue Leon talked about, but there are not many files left, so all good

File: client/src/stories/HeaderNav.stories.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import HeaderNav from '../components/HeaderNav';
+
+export default {
+  title: 'HeaderNav'
+};
+
+export function Header() {
+  return <HeaderNav></HeaderNav>;
+}
```
Comment (aruppert) [inline client/src/stories/HeaderNav.stories.js]: what a tale! ;)

File: client/src/components/HeaderNav.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <Wrapper>
+          <Img src={map}></Img>
+          <p>Karte</p>
+        </Wrapper>
+      </LeftButton>
+      <HeaderButton>
+        <Wrapper>
+          <Img src={list}></Img>
+          <p>Liste</p>
+        </Wrapper>
```
Comment (atorczynski) [inline client/src/components/HeaderNav.js]: I like the way you handled your img here. Smart!

File: client/.storybook/addons.js (hunk 1 above)
Comment (atorczynski) [reply client/.storybook/addons.js]: If you prefer double quotes on across your code -> change the .prettierrc to do so. 
This still lgtm though.

**Pull request merged**